samples/csharp_dotnetcore/12.nlp-with-luis/README.md

```diff
@@ -9,19 +9,29 @@ git clone https://github.com/Microsoft/botbuilder-samples.git
 ## Prerequisites
 ### Set up LUIS
 - Navigate to [LUIS portal](https://www.luis.ai).
+
 - Click the `Sign in` button.
+
 - Click on `My Apps`.
+
 - Click on the `Import new app` button.
+
 - Click on the `Choose File` and select [LUIS-Reminders.json](LUIS-Reminders.json) from the `botbuilder-samples/samples/csharp_dotnetcore/12.nlp-with-luis/CognitiveModels` folder.
+
 - Update [BotConfiguration.bot](BotConfiguration.bot) file with your AppId, SubscriptionKey, Region and Version. 
-    You can find this information under "Publish" tab for your LUIS application at [LUIS portal](https://www.luis.ai).  For example, for
-	https://westus.api.cognitive.microsoft.com/luis/v2.0/apps/{LuisAppID}?subscription-key={LuisSubscriptionKey}&verbose=true&timezoneOffset=0&q= 
-    
-    you will have something similar to this in the services section of your .bot file to run this sample:
+    You can find this information under "Manage" tab for your LUIS application at [LUIS portal](https://www.luis.ai).
+
+    - The `AppID` can be found in "Application Information"
+    - The `AuthoringKey` can be found in "Keys and Endpoints"
+    - The `SubscriptionKey` can be found in "Keys and Endpoints", under the `Key 1` column
+    - The `region` can be found in "Keys and Endpoints", under the `Region` column
+
+    You will have something similar to this in the services section of your .bot file to run this sample:
+
     ```
     {
```

Comment:
See if you can make the 'template' look a bit more like the change introduced in Microsoft#713 in [12.nlp-with-luis](https://github.com/southworkscom/BotBuilder-Samples/blob/master/samples/javascript_typescript/12.nlp-with-luis/README.md#set-up-luis)